healthcareai/common/model_eval.py

```diff
@@ -37,140 +51,122 @@ def clfreport(modeltype, debug, devcheck, algo, X_train, y_train, X_test, y_test
     # Initialize conditional vars that depend on ifelse to avoid PC warning
     y_pred_class = None
     y_pred = None
+    clf = algo
 
     # compare algorithms
-    if devcheck == 'yesdev':
+    if develop_model_mode is True:
         if tune:
-            clf = GridSearchCV(algo,
-                               param,
-                               cv=5,
-                               scoring='roc_auc',
-                               n_jobs=cores)
-        else:
-            clf = algo
+            # Set up grid search
+            clf = GridSearchCV(algo, param, cv=5, scoring='roc_auc', n_jobs=cores)
```

Comment:
Little confused on the flow here. If we're in the else (i.e., tune is False), then why are we using GridSearchCV?

Setting tune to False would seem to indicate no grid search is necessary.

Also, we had talked about switching to RandomizedSearchCV--still the plan?